tests/Pslayers/PslayersTest.php

```diff
@@ -251,4 +251,50 @@ public function testMasterRender()
 
         // $instance->render();
     }
+
+    //Output path was not really optional
+    public function testOptionalOutputPath()
+    {
+        $width = 830;
+        $height = 360;
+
+        $config = [
+            'id' => 11,
+            'width' => $width,
+            'height' => $height,
+        ];
+
+        $instance = new Pslayers($config);
+
+        // background layer
+        $backgroundLayer = new SolidLayer([
+            'id' => 'master-layer-solid-base',
+            'width' => $width,
+            'height' => $height,
+            'opacity' => 1.0,
+            'positionX' => 0,
+            'positionY' => 0,
+            'composite' => Imagick::COMPOSITE_DEFAULT,
+            'colour' => '#0F0',
+        ]);
+
+        $instance->addLayer($backgroundLayer, 0);
+
+        // gradient layer
+        $gradientLayer = new GradientLayer([
+            'id' => 'master-layer-gradient',
+            'width' => $width,
+            'height' => $height / 2,
+            'opacity' => 0.9,
+            'positionX' => 0,
+            'positionY' => 0,
+            'composite' => Imagick::COMPOSITE_OVER,
+            'startColour' => '#F00',
+            'endColour' => 'transparent',
+        ]);
+
+        $instance->addLayer($backgroundLayer, 0);
+        $instance->addLayer($gradientLayer, 1);
+        $instance->render();
```

Comment:
Can we add some assertions after this to ensure what we get is what we want?

Ensure the filetype, the size etc is correct.